uma/uma.py

```diff
@@ -359,6 +359,68 @@ def create_pay_request(
     )
 
 
+def create_pay_request_with_payer_data(
```

Comment:
Might be good to have the above function call this one for code deduplication purposes.